File: app/src/main/java/com/mithilakshar/mithilapanchang/mantradatamodel.java
package com.mithilakshar.mithilapanchang;

public class mantradatamodel {

    String mantraDesc,mantraName,mantraImageurl;

    public mantradatamodel() {

    }

    public mantradatamodel(String mantraDesc, String mantraName, String mantraImageurl) {
        this.mantraDesc = mantraDesc;
        this.mantraName = mantraName;
        this.mantraImageurl = mantraImageurl;
    }

    public String getMantraDesc() {
        return mantraDesc;
    }

    public void setMantraDesc(String mantraDesc) {
        this.mantraDesc = mantraDesc;
    }

    public String getMantraName() {
        return mantraName;
    }

    public void setMantraName(String mantraName) {
        this.mantraName = mantraName;
    }

    public String getMantraImageurl() {
        return mantraImageurl;
    }

    public void setMantraImageurl(String mantraImageurl) {
        this.mantraImageurl = mantraImageurl;
    }
}

File: app/src/main/java/com/mithilakshar/mithilapanchang/mantradatamodel.kt
package com.mithilakshar.mithilapanchang

class mantradatamodel {
    var mantraDesc: String? = null
    var mantraName: String? = null
    var mantraImageurl: String? = null

    constructor()
    constructor(mantraDesc: String?, mantraName: String?, mantraImageurl: String?) {
        this.mantraDesc = mantraDesc
        this.mantraName = mantraName
        this.mantraImageurl = mantraImageurl
    }
}
